.circleci/config.yml

```diff
@@ -336,12 +336,15 @@ jobs:
       - run:
           name: Save Yarn version
           command: yarn --version > /tmp/YARN_VERSION
+      - run:
+          name: Save Foundry version
+          command: node -e "process.stdout.write(require('./package.json').foundryup.version)" > /tmp/FOUNDRY_VERSION
```

Comment:
So I've added a new field to our `package.json`: `foundryup`. It contains the release we use in the installation process, as well as in this cache process.

I don't like it.

Please please please recommend a better way <3

Reply:
For caching, I just copied the way we save the yarn version (by writing to a tmp file). I don't care either way though.

What I like about using `node` where feasible, instead of *nix tools (`fromJson` in this case), is that we know that all developers at MetaMask have node installed, but they might not have `fromJson`. It makes running the commands easier.

I don't care all that much though. Just my own preference.